### ISSUE TITLE
Streamcloud filter?

### ISSUE THREAD

**Issue by Nezahual** [reek/anti-adblock-killer#3505]
http://streamcloud.eu has started to block adblocks. I'm using uBlock so what lines do I need to add and where? I'm still a bit newbie using this but I've already seen that each site needs a particular filter and I guess I have to add it to "My filters" tab, right? 

Thanks!

**Comment by uBlock-user** [comment]
```
||streamcloud.eu^$csp=style-src 'self'
streamcloud.eu##.info_text
```

**Comment by Nezahual** [comment]
Thanks! But did you tried it? It removes every ad but also breaks the player which looks like this and doesn't let me play the video.

![streamcloudaak](https://user-images.githubusercontent.com/14734857/29486186-951a228a-84e0-11e7-8557-54138b939981.png)

**Comment by uBlock-user** [comment]
Yes I did and I get a 404 when I click on the watch video now link.

**Comment by uBlock-user** [comment]
Nope, I can't reproduce what you see. I rather get 404 after I click watch video now!

**Comment by Nezahual** [comment]
I've tried it with several links and I'm getting exactly the same problem with all of them, so I guess it has nothing to do with an exact page, but anyway you can check these two as examples:

[http://streamcloud.eu/uo7w1t2acw6f/LiNkMaN_Th3.Y0ung.P0p3-VOSE-m720p_1x10.mkv.html](http://streamcloud.eu/uo7w1t2acw6f/LiNkMaN_Th3.Y0ung.P0p3-VOSE-m720p_1x10.mkv.html)
[http://streamcloud.eu/h4ia8dvrdxby/The_young_pope_-_1x10__Final__.mkv.html](http://streamcloud.eu/h4ia8dvrdxby/The_young_pope_-_1x10__Final__.mkv.html)

 =(

**Comment by uBlock-user** [comment]
I completely  disabled uBO and I still get same 404 on those links.

**Comment by Nezahual** [comment]
That's weird mate! I don't know if it could have something to do, but just to test, try only disabling uBlock Protector (in case you have it) since I don't have it installed (just uBO and AaK), I cannot think of anything more.

**Comment by uBlock-user** [comment]
patched on uBlock Protector, force an update!

**Comment by amaru7** [comment]
I have the same issue, applying the manual filter does seem to work but breaks the player here aswell. Just updated uBlock Protector and the issue still there.

**Comment by jspenguin2017** [comment]
The fix in uBlock Protector still works for me: 
![image](https://user-images.githubusercontent.com/7283682/29490856-73c93424-8506-11e7-80b1-9657cf286ec7.png)
![image](https://user-images.githubusercontent.com/7283682/29490857-8dafe414-8506-11e7-9605-db60bed49f8a.png)
The timer is validated on server side, so you have to wait.

**Comment by BobxPanda** [comment]
How do I change the command so that it works with Adblock Plus?

**Comment by amaru7** [comment]
@jspenguin2017 it doesn't with me. The time doesn't even show up, updated again, same issue.

**Comment by Nezahual** [comment]
@jspenguin2017 @uBlock-user same here. uBlock Protector installed and updated, both checkboxes marked aswell but the player is still broken. Also, to give more info about the problem, I've noticed that since I added the filters, the "waiting page" is different too, the container has now a half of its previous height and the timer and "watch video now" button are at the left of that container. Maybe uBlock blocked/deleted wrong elements and perhaps it has something to do with the broken player, this is how that "waiting page" looks now:

![streamcloudaak2](https://user-images.githubusercontent.com/14734857/29494874-ca4afd8e-85b3-11e7-8d5e-1539d68f21e5.png)

**Comment by jspenguin2017** [comment]
@amaru7 @Nezahual Configure your setup **exactly** like this and try again: https://github.com/jspenguin2017/uBlockProtector/blob/master/Notes/Standard%20Configuration.MD

**Comment by Nezahual** [comment]
@jspenguin2017 done everything suggested but nope, it didn't do the trick =(

**Comment by jspenguin2017** [comment]
@Nezahual Try resetting uBO to default settings, empty any custom rules you may have.

**Comment by Nezahual** [comment]
@jspenguin2017 nope, it didn't work either, but after several tests I have some additional info.
First, if I select the flash version of the player instead of HTML5 version, it works fine and it also remembers that preference, which is nice and would eventually "solve" the problem, a bit odd tho.
Second, about the filters given by @uBlock-user , if I remove/comment the first line (`||streamcloud.eu^$csp=style-src 'self'`), the player works and I don't have ads except for a small 'X' at the right bottom corner of the browser. That 'X' (close button) belongs to the classic pop-up saying that Sarah is near and wants to meet me.

**Comment by jspenguin2017** [comment]
I don't see any "X".

**Comment by MarcoAurelioWM** [comment]
`||streamcloud.eu^$csp=style-src 'self'`
`streamcloud.eu##.info_text`

Does not work for Adguard in my iPad. It complains that the rule is malformed. If I only leave the last line it removes the adblock warning, but cannot see the player either.

**Comment by jspenguin2017** [comment]
I heard Adguard for iOS is just a DNS server, it's not a real ad blocker.

**Comment by jspenguin2017** [comment]
@MarcoAurelioWM Maybe it has something to do with this? https://blog.adguard.com/en/adguard-ios-1-3-0/

**Comment by MarcoAurelioWM** [comment]
@uBlock-user Adguard uses reek/anti-adblock in their filters, that's why I
came here. If it does DNS filtering in the background that I don't know;
I'm not really that savy but the Adguard app for iPad does let me select
and load filters and this is one of them; so maybe it is not just DNS
filtering. Regards.
